src/components/step-flow/step-flow.test.tsx

```diff
@@ -48,211 +48,209 @@ function calculateStepStateIndexes(
-        });
-      }
-    );
-
-    it.each(generateCurrentStepOverTotalStepsVariations())(
-      "renders correct visually hidden text when 'totalSteps' prop is %s and is lower than 'currentStep' prop of %s",
-      (totalSteps, currentStep) => {
-        render(
-          <StepFlow
-            category="foo"
-            title="bar"
-            totalSteps={totalSteps}
-            currentStep={currentStep}
-            ref={() => {}}
-          />,
-          {
-            wrapper: ({ children }: { children: React.ReactNode }) => (
-              <I18nProvider locale={enGB}>{children}</I18nProvider>
-            ),
-          }
-        );
-
-        const title = screen.getByRole("heading", { level: 1 });
-        expect(
-          within(title).getByText(
-            `foo. bar. Step ${totalSteps} of ${totalSteps}.`
-          )
-        ).toHaveStyle({
-          border: "0",
-          height: "1px",
-          margin: "-1px",
-          overflow: "hidden",
-          padding: "0",
-          position: "absolute",
-          width: "1px",
-          "white-space": "nowrap",
-        });
-      }
-    );
+      const title = screen.getByRole("heading", { level: 1 });
+      expect(
+        within(title).getByText(
+          `foo. bar. Step ${currentStep} of ${totalSteps}.`
+        )
+      ).toHaveStyle({
+        border: "0",
+        height: "1px",
+        margin: "-1px",
+        overflow: "hidden",
+        padding: "0",
+        position: "absolute",
+        width: "1px",
+        "white-space": "nowrap",
+      });
+    }
+  );
 
-    it("when the 'showCloseIcon' prop is true, the correct element renders", () => {
+  it.each(generateCurrentStepOverTotalStepsVariations())(
+    "renders correct visually hidden text when 'totalSteps' prop is %s and is lower than 'currentStep' prop of %s",
+    (totalSteps, currentStep) => {
       render(
         <StepFlow
-          title="baz"
-          totalSteps={6}
-          currentStep={1}
-          showCloseIcon
+          category="foo"
+          title="bar"
+          totalSteps={totalSteps}
+          currentStep={currentStep}
           ref={() => {}}
-        />
+        />,
+        {
+          wrapper: ({ children }: { children: React.ReactNode }) => (
+            <I18nProvider locale={enGB}>{children}</I18nProvider>
+          ),
+        }
       );
 
-      expect(screen.getByLabelText("Close")).toBeVisible();
-    });
+      const title = screen.getByRole("heading", { level: 1 });
+      expect(
+        within(title).getByText(
+          `foo. bar. Step ${totalSteps} of ${totalSteps}.`
+        )
+      ).toHaveStyle({
+        border: "0",
+        height: "1px",
+        margin: "-1px",
+        overflow: "hidden",
+        padding: "0",
+        position: "absolute",
+        width: "1px",
+        "white-space": "nowrap",
+      });
+    }
+  );
+
+  it("when the 'showCloseIcon' prop is true, the correct element renders", () => {
```

Comment:
suggestion(non-blocking): language still wrong on this test

Might be worth adding to the re-visit ticket rather than dealing with it now unless there were other changes to be made